ovis-daos-sampler-rpmlintrc

```diff
@@ -0,0 +1 @@
+addFilter("devel-file-in-non-devel-package .*/usr/lib64/ovis-ldms/.*.so")
```

Comment:
This seems like an `rpmlint` bug.  This is a DSO that is clearly in a "private" dir as it should be.